mlrun/projects/project.py

```diff
@@ -2009,6 +2007,31 @@ def enable_model_monitoring(
             project=self.name,
             image=image,
             base_period=base_period,
+        )
+
+    def update_model_monitoring_controller(
+        self,
+        base_period: int = 10,
+        overwrite: bool = False,
+        image: str = "mlrun/mlrun",
+    ) -> dict:
+        r"""
```

Comment:
Why `r"""`?

Reply:
Not sure that I understand the response here. I think that the r-string is unnecessary